app/controllers/api/v0x1/root_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V0x1
+    class RootController < ApplicationController
+      def openapi
+        render :json => Rails.root.join('public', 'approval', 'v0.1.0', 'openapi.json').read
```

Comment:
We should wait until https://github.com/ManageIQ/approval-api/pull/73 is merged and update this to `v1.0.0`.

Additionally, based on https://github.com/ManageIQ/catalog-api/pull/177#discussion_r270151187 should we drop the third digit and just keep major/minor?  cc @syncrou